wmn-data.json

```diff
@@ -5165,6 +5165,16 @@
         "known" : ["missmoonified", "honey"],
         "cat" : "xx NSFW xx"
        },
+       {
+        "name": "Substack",
+        "uri_check": "https://substack.com/@{account}",
+        "e_code" : 200,
+        "e_string" : "Subscribe",

```

Comment:
Can you choose a different `e_string` please? I am finding the response includes `subscribe` in both true and false cases. Please also pick a more specific `m_string` as the number 404 is much too generic.

Suggested `e_string` = `| Substack</title>`
Suggested `m_string` = `&quot;on Substack</title>`

Reply:
Yes, you have right. Sorry for this.

i think here should be space between '&quot;' and 'on'  `m_string = &quot; on Substack</title>`

<title>Search "invalid" on Substack</title>